Tests: test fails with go1.15

**Describe the bug**

Since go1.15, there's a new vet check for code such as `string(x)` where x has an integer type other than `rune`.
This vet check is enabled by default on `go test`. https://golang.org/doc/go1.15#vet

**To Reproduce**

Steps to reproduce the behavior:
1. use go v1.15
2. `go test ./pkg/chunkenc/`

**Expected behavior**
tests pass

**Environment:**
 - go v1.15
 - linux

**Screenshots, Promtail config, or terminal output**

```
 aurelien-> go test ./pkg/chunkenc/
# github.com/grafana/loki/pkg/chunkenc
pkg/chunkenc/memchunk_test.go:258:59: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
pkg/chunkenc/memchunk_test.go:274:22: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
FAIL    github.com/grafana/loki/pkg/chunkenc [build failed]
FAIL
```

--- a/pkg/chunkenc/memchunk_test.go
+++ b/pkg/chunkenc/memchunk_test.go
@@ -7,6 +7,7 @@ import (
 	"fmt"
 	"math"
 	"math/rand"
+	"strconv"
 	"strings"
 	"testing"
 	"time"
@@ -255,7 +256,7 @@ func TestSerialization(t *testing.T) {
 			numSamples := 50000
 
 			for i := 0; i < numSamples; i++ {
-				require.NoError(t, chk.Append(logprotoEntry(int64(i), string(i))))
+				require.NoError(t, chk.Append(logprotoEntry(int64(i), strconv.Itoa(i))))
 			}
 
 			byt, err := chk.Bytes()
@@ -271,7 +272,7 @@ func TestSerialization(t *testing.T) {
 
 				e := it.Entry()
 				require.Equal(t, int64(i), e.Timestamp.UnixNano())
-				require.Equal(t, string(i), e.Line)
+				require.Equal(t, strconv.Itoa(i), e.Line)
 			}
 			require.NoError(t, it.Error())
 

